python/.openapi-generator/FILES

```diff
@@ -1,4 +1,236 @@
+.github/workflows/python.yml
+.gitignore
+.gitlab-ci.yml
+.openapi-generator-ignore
+.travis.yml
 README.md
+docs/AddCollection200Response.md
```

Comment:
are this files here correct? you delete the docs in the generate.py right?

Reply:
This folder is entirely useless. I just deleted it: [8a54d46](https://github.com/geo-engine/openapi-client/pull/16/commits/8a54d462ae41d81eacd57909bd3fca19039f4e26)